libraries/chain/platform_timer_asio_fallback.cpp

```diff
@@ -56,11 +56,14 @@ platform_timer::~platform_timer() {
 }
 
 void platform_timer::start(fc::time_point tp) {
+   assert(_state == state_t::stopped);
    if(tp == fc::time_point::maximum()) {
       _state = state_t::running;
+      timer_running_forever = true;
       return;
    }
    fc::microseconds x = tp.time_since_epoch() - fc::time_point::now().time_since_epoch();
+   timer_running_forever = false;
```

Comment:
No need to change, just a remark. Personally I prefer initializing members once if possible, as in:
```suggestion
   timer_running_forever = tp == fc::time_point::maximum();
   if(timer_running_forever ) {
      _state = state_t::running;
      return;
   }
   fc::microseconds x = tp.time_since_epoch() - fc::time_point::now().time_since_epoch();
```

Reply:
that does look good here